src/Sql/Sql.Test/UnitTests/PermissiveRecordMatcherWithApiExclusionUnitTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.WindowsAzure.Commands.ScenarioTest;
```

Comment:
@johnpaulkee TBD on this file 😁

Reply:
Acked 👍 I'm hoping we can come to a solution sometime this week.